docs/cloud/index.md

```diff
@@ -64,13 +64,13 @@ Now you can start running your tests. Use the following command to execute the C
 <TabItem value="iOS" label="iOS">
 
 ```shell
-marathon-cloud -api_key generated_api_key -apk sample.zip -testapk sampleUITests-Runner.zip -platform iOS
+marathon-cloud run ios --api_key generated_api_key --application sample.zip --test-application sampleUITests-Runner.zip
```

Comment:
I'd add a section describing default way of doing this via envvar `MARATHON_CLOUD_API_KEY` that is in the --help of run commands. As an alternative `--api-key` (note the `-` instead of `_`) can be used